test/resources/sample.test_feedback.sh

```diff
@@ -16,11 +16,18 @@ echo "Feeding test results back to Prout..."
 # Display expanded commands
 set -ex
 
-# Prout API endpoint
+# Prout's Travis webhook
 PROUT_HOOK=https://my-prout-host/api/hooks/travis
 
+# SauceLabs session ID
+if [ ! -f ./logs/screencastId ]; then
+    SCREENCAST_ID=""
+else
+    SCREENCAST_ID=`cat ./logs/screencastId`
```

Comment:
Example of implementation: https://github.com/mario-galic/sandbox/blob/da9e53455/test/acceptance/Config.scala#L46